test/test_enclave_wrangler/test_objects_api.py

```diff
@@ -211,3 +233,206 @@ def test_get_concept_set_version_members(self):
         obj_id = 563193300
         data = get_concept_set_version_members(obj_id, return_detail='full')
         self.assertGreater(len(data), 0)
+
+
+# TODO: new tests by claude to try
+# from pprint import pprint as pp
+# todo: #1: Wherever I put '#1', consider removing the mock and call the actual function
+class TestObjectsApiMocks(DbRefreshTestWrapper):
```

Comment:
New class that [Claude made](https://claude.ai/chat/7f64ea42-7ab2-43c6-9a86-382bdafc26bf) to do mock API tests.